console-subscriber/src/aggregator/mod.rs

```diff
@@ -489,49 +499,47 @@ impl Aggregator {
             None
         };
 
+        let new_poll_ops = if !self.new_poll_ops.is_empty() {
+            std::mem::take(&mut self.new_poll_ops)
+        } else {
+            ShrinkVec::default()
+        };
```

Comment:
nit: i think this could be simplified to just:
```suggestion
        let new_poll_ops = std::mem::take(&mut self.new_poll_ops);
```

if it's empty, `take`ing it will still just give us an empty vector, so it's fine.